patterns/1-initial/walk-the-innersource-talk.md

```diff
@@ -48,16 +49,17 @@ are often used to working behind closed doors. Decision-making happens in meetin
 scattered, and ownership is unclear. Shifting to transparent, open collaboration requires both cultural change and a
 reevaluation of existing workflows.
 - People in central roles may fear scrutiny, overload from unsolicited input, or loss of control. Additionally, moving
-towards open collaboration demands more discipline in documentation, async communication, and community
-management - skills not always prioritized in internal tooling teams.
+towards open collaboration demands more discipline in documentation, async communication, and community management -
+skills not always prioritized in internal tooling teams.
 
 ### What are the trade-offs?
 
 - **Transparency vs. Control**: sharing decisions and code openly invites feedback and builds trust, but also requires a
 mindset shift—accepting discussion, disagreement, and the need to justify choices in public forums.
```

Comment:
maybe we can mention the higher acceptance of transparent decision making as people have been involved

Reply:
Point added!